app/appshell/app.js

```diff
@@ -210,8 +211,8 @@ app.getRemoteDebuggingPort = function () {
 };
 
 app.getUserDocumentsDirectory = function () {
-    // TODO: implement
-    throw new Error("app.getUserDocumentsDirectory not implemented");
+    var _path = path.resolve(electronApp.getPath("home"), "Documents");
```

Comment:
@ingorichter I don't know how it's handled by OS X, but on Windows `Documents` is symlinked (or similar) to `Dokumente`.
That's also how it's supposed to work.